HTML/index.html

```diff
@@ -6,25 +6,32 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="description" content="Delicious' Chocolate Pizza recipe">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald:b' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Abel' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Patrick+Hand' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Droid+Serif' rel='stylesheet' type='text/css'>
+
+
   <link href="style.css" type="text/css" rel="stylesheet"/>
   <link href="960.css" type="text/css" rel="stylesheet"/>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 </head>
 
 <body>
-  <div class="container_12 test">
-
+  <div class="container_16 test">
     <header class="masthead test">
       <section class="clearfix test">
-        <div class="grid_6 test">
+        <div class="grid_8 test">
           <img class="logo" src="file:///Users/jkoehnke/Downloads/challenge/logo.png" alt="Delicious">
```

Comment:
Remember with images to not be this specific about where they are. If you put index.html back in the root directory, the src path would be:
```
<img src="img/logo.png" alt="Delicious">